### ISSUE TITLE
[Feature Request] Critical damage and chance attribute

### ISSUE THREAD

**Issue by Lix3nn53** [Eniripsa96/SkillAPI#376]
First, thanks for the updates really amazing work <3

I think you already know what I mean from title but I will explain anyways.

You normal attacks and skills deal [Critical Damage]% more damage with [Critical Chance]% chance

Also a critical-damage Trigger would be nice so we can play particles/sounds etc to let player know/feel the critical strike

Thanks for the updates again, good luck!

**Comment by Tiplish** [comment]
I have ask for critical trigger to Eniripsa #279 and he said that it's not possible :(

**Comment by Lix3nn53** [comment]
Yes its not possible for vanilla critical. But I am asking for a custom critical mechanic.

**Comment by Tiplish** [comment]
A critical chance like in MMORPG?
